### DISCUSSION TITLE
관리자 기능 서버 어플리케이션 개발 하기 전 필독!

### DISCUSSION THREAD

**Discussion by myrhymetree** [sixes-sense/admin#1] [Announcements]
**관리자 서버는 일반 사용자 용 백엔드 api 서버에서 패키지 구조를 그대로 긁어 와서 적용하였습니다.
git에 올릴 때 패키지를 그대로 유지하기 위해서 기존에 작성된 클래스를 제거하지 않았습니다.
작업하실 때 이런 부분 잘 참고해서 수정 or 제거하셔서 개발하시면 됩니다.**

주의사항
1. 로컬 데이터베이스 생성되어 있는지 확인하기
   없으면 일반사용자 백엔드 서버를 키시거나, 수동으로 만들어주시면 됩니다
   (ddl-auto 생성구문 확인해서 그대로 긁어서 복붙하시면 됩니다)

**Comment by myrhymetree** [comment]
@kny3037 @bbbbooo @ChoMyeongHwan @taegeun-park0525 
참고하셔서 개발하시면 됩니다.

**Comment by bbbbooo** [comment]
확인했습니다.

**Comment by kny3037** [comment]
확인했습니다. 감사합니다!!
